operator/internal/manifests/openshift/alertingrule.go

```diff
@@ -0,0 +1,34 @@
+package openshift
+
+import lokiv1 "github.com/grafana/loki/operator/apis/loki/v1"
+
+func AlertingRuleTenantLabels(ar *lokiv1.AlertingRule) {
+	switch ar.Spec.TenantID {
+	case tenantApplication:
+		tenantLabels := map[string]string{
+			opaDefaultLabelMatcher: ar.Namespace,
+		}
+
+		for groupIdx, group := range ar.Spec.Groups {
+			group := group
+			for ruleIdx, rule := range group.Rules {
+				rule := rule
+				for key, value := range tenantLabels {
+					if rule.Labels == nil {
+						rule.Labels = map[string]string{}
+					}
+
+					rule.Labels[key] = value
+				}
+				group.Rules[ruleIdx] = rule
+			}
+			ar.Spec.Groups[groupIdx] = group
+		}
```

Comment:
Why use a map with only one element? In this particular case we only want to use the namespace label.
```suggestion
	case tenantApplication:
		for groupIdx, group := range ar.Spec.Groups {
			group := group
			for ruleIdx, rule := range group.Rules {
				rule := rule
				if rule.Labels == nil {
					rule.Labels = map[string]string{}
				}
				rule.Labels[opaDefaultLabelMatcher] = ar.Namespace
				group.Rules[ruleIdx] = rule
			}
			ar.Spec.Groups[groupIdx] = group
		}
```